src/app/public/src/modules/sidebar/sidebar-wrapper.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="stache-sidebar-wrapper" [ngClass]="{'sidebar-closed': sidebarClosed, 'sidebar-open': !sidebarClosed  }">
+  <div class="stache-sidebar-header">
+    <button *ngIf="!sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="closeSidebar()" aria-label="sidebar close">
+      <i class="fa fa-chevron-left" aria-hidden="Close"></i>
+    </button>
+    <button *ngIf="sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="openSidebar()" aria-label="sidebar open">
```

Comment:
The aria-label here (and for the close button) would probably benefit from a clearer message: "Click to show sidebar" or something.